apps/video/js/video.js

```diff
@@ -136,6 +136,22 @@ document.addEventListener('visibilitychange', function visibilityChange() {
   }
 });
 
+// Handle interruption from audio channel and set correct
+// play/pause button. See Bug 121457
+dom.player.addEventListener('mozinterruptbegin', function() {
+  if (playing) {
+    setButtonPaused(true);
+    playing = false;
+  }
+});
+
+dom.player.addEventListener('mozinterruptend', function() {
+  if (!playing) {
```

Comment:
On further investigation playing variable is not a correct reflection of interrupted state and that's why I have removed it in the updated patch to keep patch simple to changing button play/pause icon ,  See https://bugzilla.mozilla.org/show_bug.cgi?id=1214157#c19.